.github/workflows/slash-commands.yml

```diff
@@ -0,0 +1,15 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Slash Command Dispatch
```

Comment:
does this implicitly create a `chaos-test-command` dispatch event?

Reply:
Yes. The `-command` part is configurable. Event take the name of the command + a suffix.